app/views/fs/messages/_delivery_report.html.erb

```diff
@@ -0,0 +1,88 @@
+<style>
+  th {
+    font-weight: normal;
+    text-align: left;
+  }
+  th, td {
+    padding: 0.25rem 2rem 0.25rem 0;
+  }
+</style>
+
+<table class="min-w-full divide-y divide-gray-300">
+  <tbody class="divide-y divide-gray-200">
+  <tr>
+    <th>Typ dokumentu:</th>
+    <td>
+      <b><%= message.title %></b>
+    </td>
+  </tr>
+  <tr>
+    <th>Daňový subjekt:</th>
+    <td>
+      <b><%= message.recipient_name %></b>
+    </td>
+  </tr>
+  <tr>
+    <th>Identifikátor subjektu:</th>
+    <td>
+      <b><%= message.metadata['dic'] %></b>
+    </td>
+  </tr>
+  <tr>
+    <th>Odosielateľ podania:</th>
+    <td>
+      <b><%= message.metadata['fs_submitting_subject'] %></b>
+    </td>
+  </tr>
+  <tr>
+    <th>Evidenčné číslo podania:</th>
+    <td>
+      <b><%= message.metadata['fs_sent_message_id'] %></b>
+    </td>
+  </tr>
+  <tr>
+    <th>Evidenčné číslo potvrdenia:</th>
+    <td>
+      <b><%= message.metadata['fs_message_id'] %></b>
+    </td>
+  </tr>
+  <tr>
+    <th>Dátum a čas prijatia EP:</th>
+    <td>
+      <b>
+        <time class="md:hidden" datetime="<%= message.metadata['fs_submission_created_at'] %>">
+          <%= ApplicationController.helpers.nice_datetime_with_time(Time.parse(message.metadata['fs_submission_created_at']), keep_year: true) %>
+        </time>
+      </b>
+    </td>
+  </tr>
+  <tr>
+    <th>Dátum a čas vystavenia potvrdenky:</th>
+    <td>
+      <b>
+        <time class="md:hidden" datetime="<%= message.delivered_at %>">
+          <%= ApplicationController.helpers.nice_datetime_with_time(message.delivered_at, keep_year: true) %>
+        </time>
+      </b>
+    </td>
+  </tr>
+  <tr>
+    <th>Obdobie:</th>
+    <td>
+      <b><!--TODO zistit co tu byva vyplnene (zatial sme videli iba prazdne)--></b>
```

Comment:
Toto asi bude potrebne overit s nejakym vacsim FS userom, nech zistime, co tu zvykne byt vyplnene.

Reply:
@celuchmarek Jano spominal, ze k tomuto by sme mozno vedeli vycitat nejake info aj z XSDciek k response/WSDL. Vedel by si skusit popatrat?

Reply:
V response toho môže byť viac. Všetky fieldy sú tam optional. Teraz máme v tom api iba tie, čo som dostával pri volaní, viem tam ale pomerne ľahko dohodiť všetky. Napr. obdobie je v popise API, takže pridáme do FS API a zistíme, či to aj z FS reálne vracia.